tests/csm/rest/test_csm_users.py

```diff
@@ -431,34 +446,32 @@ def test_5001(self):
     def test_5002(self):
         """
         Test that GET API with no value for sort_by param returns 400 response code
-
         """
         test_case_name = cortxlogging.get_frame()
         self.log.info("##### Test started -  %s #####", test_case_name)
         test_cfg = self.csm_conf["test_5002"]["response_msg"]
         resp_error_code = test_cfg["error_code"]
         resp_msg_id = test_cfg["message_id"]
         resp_data = self.rest_resp_conf[resp_error_code][resp_msg_id]
-        msg = resp_data[0]
-
+        resp_msg_index = test_cfg["message_index"]
+        msg = resp_data[resp_msg_index]
         self.log.info("Fetching csm user with empty sort by string...")
         response = self.csm_obj.list_csm_users(
             expect_status_code=const.BAD_REQUEST,
             sort_by="",
             return_actual_response=True)
-
+        self.log.info("Verifying the response status code")
+        assert_utils.assert_equals(response.status_code, const.BAD_REQUEST)
```

Comment:
Above comment applicable

Reply:
will have a ticket to remove where we used commons\constants under our rest api tests